### ISSUE TITLE
Directory messup

### ISSUE THREAD

**Issue by waja** [waja/maldetect#13]
There are directories directly in /

quarantine/
sess/
sigs/
sigs.old/

**Comment by waja** [comment]
actually it looks like a leftover from earlier development package.

**Comment by waja** [comment]
seems "$var_path" is not defined
```
$ rgrep var_path /etc/maldetect/
/etc/maldetect/internals.conf:quardir="$var_path/quarantine"
/etc/maldetect/internals.conf:sessdir="$var_path/sess"
/etc/maldetect/internals.conf:sigdir="$var_path/sigs"
/etc/maldetect/internals.conf:cldir="$var_path/clean"
/etc/maldetect/internals.conf:userbasedir="$var_path/pub"
```

**Comment by waja** [comment]
Should be fixed at least with 933bb7f524700a49135846dc4099a9cd5c8c7dab
